homeassistant/components/vicare/sensor.py

```diff
@@ -217,17 +225,90 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
         device_class=SensorDeviceClass.TEMPERATURE,
     ),
     ViCareSensorEntityDescription(
-        key=SENSOR_COLLECTOR_TEMPERATURE,
```

Comment:
Please see my comment in  https://github.com/home-assistant/core/issues/65633#issuecomment-1030637318
It seems this sensor does not deliver the data a user would expect here.
Can you please remove it. It should be unnecessary to provide this historic data since Home Assistant's database will track the other TOTAL_INCREASING sensors and give us the same information.

On a side-note: Changing the key is problematic since it is supposed to be a unique identifier. This will most likely create duplicated entities when people update